Hazel/src/Hazel/Log.h

```diff
@@ -4,6 +4,8 @@
 #include "spdlog/spdlog.h"
 #include "spdlog/fmt/ostr.h"
 
+#include "ImGui/ImGuiConsole.h"
```

Comment:
Why does this include the ImGuiConsole here? Is that necessary for this header file?